sonic_platform_base/chassis_base.py

```diff
@@ -46,6 +46,9 @@ class ChassisBase(device_base.DeviceBase):
     # available on the chassis
     _sfp_list = []
 
+    # List of component names that available on the chassis
+    _component_list = []
```

Comment:
`_component_name_list`, since this only holds strings, unlike the other lists which hold objects?